docs/release-notes/8.0/8.0.30-23.0.md

```diff
@@ -0,0 +1,58 @@
+# Percona XtraBackup 8.0.30-23 (2022-10-)
+
+<style>
+    table {
+        width=50%';
+        font-family: Chivo, Colfax, Franziska, Helvetica, Arial, sans-serif;
+    }
+    table td {
+        border: 0px;
+        padding: 8px;
+    }
+    tr:nth-child(even){
+        background-color:#f5f5f5
+    }
+    tr:hover {
+        background-color: #dddd;
+    }
+</style>
+
+| Release date | October 2022 |
+|---|---|
+| Install instructions | [Install Percona XtraBackup](https://www.percona.com/doc/percona-xtrabackup/8.0/installation.html) |
+
+[Percona XtraBackup for MySQL Databases](https://www.percona.com/software/mysql-database/percona-xtrabackup) enables MySQL backups without blocking user queries. Percona XtraBackup is ideal for companies with large data sets and mission-critical applications that cannot tolerate long periods of downtime. Offered free as an open source solution, Percona XtraBackup drives down backup costs while providing unique features for MySQL backups. [Commercial support contracts are available](https://www.percona.com/services/support/mysql-support/).
+
+## Release highlights
+
+New features and improvements introduced in Percona XtraBackup 8.0.30-23:
+
+* Percona XtraBackup adds support for [`Zstandard (ZSTD)`](/docs/backup_scenarios/compressed_backup.md#creating-compressed-backups) compression algorithm. `ZSTD` is a fast lossless compression algorithm, targeting real-time compression scenarios at zlib-level and better compression ratios. 
+
+* Percona XtraBackup implements the `Smart memory estimation` feature to compute the memory required to `--prepare` a backup. Percona XtraBackup has extended the crash recovery logic to extract the formula used to allocate memory. Now, in the backup phase, while copying redo log entries, Percona XtraBackup computes the required memory for `--prepare`. Percona XtraBackup also takes into consideration the number of InnoDB pages to be fetched from the disk. Percona XtraBackup checks the server's available free memory and uses that memory up to the limit specified in the `--use-free-memory-pct` parameter to run `--prepare`.
+
+## New features
+
+[PXB-2669](https://jira.percona.com/browse/PXB-2669): Implements support for `Zstandard (ZSTD)` compression algorithm.
+
+## Improvements
+
+[PXB-2710](https://jira.percona.com/browse/PXB-2710): Implements support for `Smart Memory Estimation`.
+[PXB-2844](https://jira.percona.com/browse/PXB-2844): Adjusts the logic to detect multi-threaded slave.
+
+## Bugs fixes
+
+[PXB-2681](https://jira.percona.com/browse/PXB-2681): Percona XtraBackup got stuck during incremental backup prepare of encrypted tables.
+[PXB-2517](https://jira.percona.com/browse/PXB-2517): Using innobackupex for database backup failed.
+[PXB-2702](https://jira.percona.com/browse/PXB-2702): Backup failed when undo truncation log is present.
```

Comment:
Add a note:
Thank you Dmitry Smal for your contribution for initial patch.

Reply:
Updated